procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    logger = logger.getLogger("Daytime Checkout"+f" {Time.now()} UT")
</pre><p>What is that f?</p><p>if you want to use an f-string then it's </p><pre class="ql-syntax" spellcheck="false">logger = logger.getLogger(f"Daytime Checkout {Time.now()} UT")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes. That's an f string in the middle of the string, not very elegant, I admit. Corrected.